src/SampleLibraryTests/HelloDynamoZeroTouchTests.cs

```diff
@@ -44,13 +44,15 @@ namespace SampleLibraryTests
     class HelloDynamoZeroTouchTests : GeometricTestBase
     {
         [Test]
+        [Category("NEEDS_ASM")]
```

Comment:
I am not sure we should call this ASM - maybe geo lib or something less ADSK speak.